src/operators/ai.onnx/Add/7/execute_operator__ai_onnx__add__7__T_tensor_uint64.c

```diff
@@ -0,0 +1,39 @@
+//this file was generated by ../../../../../../scripts/onnx_generator/OperatorTemplate.py
+#include "operator__ai_onnx__add__7.h"
+#include "tracing.h"
+#include "utils.h"
+
+operator_status
+execute_operator__ai_onnx__add__7__T_tensor_uint64(
+    node_context *ctx
+)
+{
+    TRACE_ENTRY(1);
+
+    TRACE_NODE(2, true, ctx->onnx_node);
+
+    /* UNCOMMENT AS NEEDED */
+
+    // Onnx__TensorProto *A = searchInputByName(ctx, 0);
```

Comment:
Since I split up the function I don't have the problem anymore of duplicate names, but I noticed that attributes, input & output tensors can have the name! see [BatchNormalization](https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization). maybe we should go with these prefixes: `i_`, `o_`, `a_` for input/output tensor and attributes?

Reply:
I introduced the prefixes